test/e2e/multihoming_utils.go

```diff
@@ -520,3 +520,18 @@ func getNetworkGateway(cli *client.Client, networkName string) (string, error) {
 
 	return "", fmt.Errorf("Gateway not found for network %q", networkName)
 }
+
+// in order to ensure that Network Attached Definitions are cleaned up properly the pods must be removed first
+func cleanupPods(cs clientset.Interface, namespace string) error {
```

Comment:
you can move this to network_segmentation.go itself since this isn't exactly part of multi homing feature

Reply:
moved